bindings/nodejs/examples/how_tos/nft_collection/01_mint_collection_nft.ts

```diff
@@ -0,0 +1,116 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import { MintNftParams, NftId, utf8ToHex, Utils, Wallet } from '@iota/sdk';
+require('dotenv').config({ path: '.env' });
+
+// The NFT collection size
+const NFT_COLLECTION_SIZE = 10000;
```

Comment:
You're crazy :carlos:
```suggestion
const NFT_COLLECTION_SIZE = 150;
```